src/frontend/src/pages/Calendar/CalendarComponents/CalendarDayCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Card, CardContent, Grid, IconButton, Typography } from '@mui/material';
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
+import { DesignReview } from 'shared';
+import { meetingStartTimePipe } from '../../../utils/pipes';
+
+interface CalendarDayCardProps {
+  myDate: Date;
+  events: DesignReview[];
+}
+
+const CalendarDayCard: React.FC<CalendarDayCardProps> = ({ myDate, events }) => {
+  const DayCardTitle = () => (
+    <Grid container alignItems="center" margin={0} padding={0}>
+      <Grid item>
+        <IconButton>
+          <AddCircleOutlineIcon fontSize="small" />
+        </IconButton>
+      </Grid>
+      <Grid item xs display="flex" justifyContent="flex-end">
+        <Typography variant="h6" marginRight={1}>
+          {myDate.getDate()}
+        </Typography>
+      </Grid>
+    </Grid>
+  );
+
+  const EventCard = (event: DesignReview) => {
+    const name = event.designReviewId;
+    return (
+      <Box marginLeft={0.5}>
+        <Card sx={{ backgroundColor: 'red', borderRadius: 1, minWidth: 140, maxWidth: 140, minHeight: 20, maxHeight: 20 }}>
+          <Typography marginLeft={0.5} marginBottom={0.3}>
+            {name + ' ' + meetingStartTimePipe(event.meetingTimes)}
```

Comment:
you should be displaying an icon here based on the team type (this will be from an iconName tracked in the model for a design review once we add it but it can be hardcoded for now):
<img width="71" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/1b0fa84e-920d-45ed-97f4-3df224a1177b">